main.go

```diff
@@ -16,6 +16,70 @@
 
 package main
 
+import (
+	"cmd-exclude-prefixes-k8s/internal/prefixcollector"
+	"cmd-exclude-prefixes-k8s/internal/utils"
+	"context"
+
+	"github.com/kelseyhightower/envconfig"
+	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/networkservicemesh/sdk/pkg/tools/jaeger"
+	"github.com/networkservicemesh/sdk/pkg/tools/signalctx"
+	"github.com/networkservicemesh/sdk/pkg/tools/spanhelper"
+)
+
+// Config - configuration for cmd-exclude-prefixes-k8s
+type Config struct {
+	ExcludedPrefixes   []string             `desc:"List of excluded prefixes" split_words:"true"`
+	ConfigMapNamespace string               `default:"default" desc:"Namespace of kubernetes config map" split_words:"true"`
+	KubeConfig         utils.KubeConfigPath `desc:"Path to kubeconfig file"`
```

Comment:
I think actually we do not need this. Use KUBECONFIG env is natural and it is not related to Exclude Prefixes service.

Reply:
Removed from config.